client/src/utils/game-channel/game-channel.ts

```diff
@@ -638,7 +638,10 @@ export class GameChannel {
       responderId: this.channelConfig.responderId,
     };
 
-    const message = JSON.stringify(data);
+    const url = 'https://statechannel.aepps.com/rock-paper-scissors';
+    const message = `I just played a game of rock-paper-scissors on the æternity blockchain. Try it on ${url} | ${JSON.stringify(
+      data
```

Comment:
Wouldn't it be more clear to have the data in 2 seperate fields? Since we `JSON.stringify` it, it's possible to have the text and the stats in 2 different fields, similar to how we use `localStorage`

Reply:
That way we avoid _`split`ting_ on reading as well

Reply:
Then it would look weird on the explorer

Reply:
I think that is dependent on which explorer we will use, since a new one is under development, we could provide them with some feedback about supporting json information :thinking: